powerauth-fido2/src/main/java/com/wultra/powerauth/fido2/rest/model/converter/serialization/CollectedClientDataDeserializer.java

```diff
@@ -60,7 +60,8 @@ public CollectedClientData deserialize(JsonParser jsonParser, DeserializationCon
             collectedClientData.setEncoded(new String(decodedClientDataJSON));
             return collectedClientData;
         } catch (IOException e) {
-            throw new RuntimeException(e);
+            logger.warn(e.getMessage(), e);
```

Comment:
dtto

Reply:
I will leave the implementation as is for now, because I do not think a generic error handler for `IOException` is a good idea and we want to avoid unhandled 500 errors. We can improve it later in case of any issues, however this is a corner case which should not occur in actual implementation because the collected client data object is provided by the authenticator and should be serialized correctly.